error while "Running a Build" on linux

### Describe the bug

[Running a build](https://github.com/microsoft/msquic/blob/main/docs/BUILD.md#running-a-build-1) produces error in 
```
PS /home/mininet/msquic/build> uname -a
Linux mininet-vm 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

### Steps to reproduce the behavior

1. `PS /home/mininet/msquic/build> cmake --build .`
2. View output with error
```
[ 31%] Built target core
[ 31%] Built target mkdir_openssl_build
[ 31%] Built target OpenSSL_Build
[ 32%] Building C object src/platform/CMakeFiles/platform.dir/datapath_epoll.c.o
/home/mininet/msquic/src/platform/datapath_epoll.c: In function 'CxPlatSocketConfigureRss':
/home/mininet/msquic/src/platform/datapath_epoll.c:890:13: error: 'SO_ATTACH_REUSEPORT_CBPF' undeclared (first use in this function)
  890 |             SO_ATTACH_REUSEPORT_CBPF,
      |             ^~~~~~~~~~~~~~~~~~~~~~~~
/home/mininet/msquic/src/platform/datapath_epoll.c:890:13: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [src/platform/CMakeFiles/platform.dir/build.make:63: src/platform/CMakeFiles/platform.dir/datapath_epoll.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:635: src/platform/CMakeFiles/platform.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

### Expected vs actual behavior

Expected to run successfully

--- a/src/platform/datapath_epoll.c
+++ b/src/platform/datapath_epoll.c
@@ -863,7 +863,6 @@ Exit:
     return Status;
 }
 
-
 QUIC_STATUS
 CxPlatSocketConfigureRss(
     _In_ CXPLAT_SOCKET_CONTEXT* SocketContext,

--- a/src/platform/datapath_epoll.c
+++ b/src/platform/datapath_epoll.c
@@ -863,12 +863,14 @@ Exit:
     return Status;
 }
 
+
 QUIC_STATUS
 CxPlatSocketConfigureRss(
     _In_ CXPLAT_SOCKET_CONTEXT* SocketContext,
     _In_ uint32_t SocketCount
     )
 {
+#ifdef SO_ATTACH_REUSEPORT_CBPF
     QUIC_STATUS Status = QUIC_STATUS_SUCCESS;
     int Result = 0;
 
@@ -901,6 +903,9 @@ CxPlatSocketConfigureRss(
     }
 
     return Status;
+#else
+    return QUIC_STATUS_NOT_SUPPORTED;
+#endif
 }
 
 //

